source/features/linkify-user-location.tsx

```diff
@@ -3,47 +3,35 @@ import select from 'select-dom';
 import {wrap} from '../libs/dom-utils';
 import features from '../libs/features';
 
-function addLocation(baseElement?: HTMLElement): void {
-	const locationIcon = select('.octicon-location', baseElement)!;
-
-	if (!locationIcon) {
+function addLocation(baseElement: HTMLElement): void {
+	const location = select('.octicon-location', baseElement)?.nextSibling;
```

Comment:
This doesn't work, location elements in hovercards and in profiles have different DOM structure - in profiles location text is inside a <span>, in hovercards it's just a textnode.

Reply:
Right, nextSibling works for any node: Element and TextNode.

Reply:
And because of that, on user profile page it picks whitespace node right after the icon and linkifies it

Reply:
The linkifying needs to be improved, true. Even earlier it would also linkify the space before the location, so the underline extended all the way to the icon